common/djangoapps/student/admin.py

```diff
@@ -50,10 +50,18 @@
 
 User = get_user_model()  # pylint:disable=invalid-name
 
-# This switch exists because the CourseEnrollment admin views make DB queries that impact performance.
-# In a large enough deployment of Open edX, this is enough to cause a site outage.
 # See https://openedx.atlassian.net/browse/OPS-2943
-COURSE_ENROLLMENT_ADMIN_SWITCH = LegacyWaffleSwitch(STUDENT_WAFFLE_NAMESPACE, 'courseenrollment_admin', __name__)
+# .. toggle_name: student.courseenrollment_admin
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: This toggle will enable the rendering of the admin view of the CourseEnrollment model..
+# .. toggle_warnings: Enabling this toggle may impact in the performance of the platform.
```

Comment:
Either update to something like: "...may adversely impact...", or something simpler like: "Enabling this toggle may cause performance problems."